tools/test_storage_hyperv_primitive.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python3
+
+import json
+import sys
+
+from cloudcix_primitives import storage_hyperv
+
+cmd = sys.argv[1]
+
+host = None
+domain_path = 'D:\HyperV\primitive_test\\'
+storage = 'primitive_test.vhdx'
+size = 5
+update_size = 6
+
+if len(sys.argv) > 2:
+    host = sys.argv[2]
+
+if len(sys.argv) > 3:
+    storage = sys.argv[3]
+
+if len(sys.argv) > 4:
+    size = sys.argv[4]
+
+if len(sys.argv) > 5:
```

Comment:
This arg can be removed, `cloudimage` is not a part of the contract for storage_hyperv